src/server/utils/asynctaskexecutor.h

```diff
@@ -27,8 +23,8 @@ class AsyncTaskExecutor {
     static AsyncTaskExecutor* inst;
 };
 
-#define ASYNC_TASK(task) \
-    utils::AsyncTaskExecutor::instance()->submit_task(task);
+/*#define ASYNC_TASK(task) \
+    utils::AsyncTaskExecutor::instance()->submit_task(task);*/
```

Comment:
Remove this macro if we don't need it anymore

Reply:
done

Reply:
Good boie